devel/release.sh

```diff
@@ -116,14 +120,12 @@ inform Checkout $VERSION
 git checkout $VERSION
 if [ $? -ne 0 ]; then echo "Failed checking our $VERSION"; exit 2; fi
 
-inform Build
-make
```

Comment:
This seems to be a mix up with a previous commit. Shouldn't this be part of "remove confusing makefile"?

Reply:
Yeah...it's a sign I did not run test tests properly